app/serializers/background_serializer.rb

```diff
@@ -0,0 +1,6 @@
+class BackgroundSerializer
+  include FastJsonapi::ObjectSerializer
+
+  set_type 'background'
+  link :url
```

Comment:
Adding [:link] to the serializer was new to me here - will definitely remember for future projects.